ccp/modules/mtba-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: "3.7"
-
```

Comment:
Why is this removed?

Reply:
It is already set in the root docker-compose.yml

Reply:
Version 3.7. ist back again